### DISCUSSION TITLE
MCMC run of DESIJ0618+5018

### DISCUSSION THREAD

**Discussion by aaimtiaz** [AstroBridge/BDLensing#261] [General]
I have run almost 1600 steps. Parameters haven't converged yet. These are the visuals:
![image](https://github.com/ajshajib/BDLensing/assets/78560474/2bba292c-5a30-4f87-abac-9a799bd04ab8)
![image](https://github.com/ajshajib/BDLensing/assets/78560474/029e8c19-82de-4784-a204-9713c30662d0)

After the MCMC run, the blue residual at 5 o'clock got darker. Except for this, other residuals improved. Should I mask this? @ajshajib vai?

And another thing is that there are some low-flux regions (black) in the reconstructed source light; should I be worried about these? @ajshajib vai?

**Comment by aaimtiaz** [comment]
@SulymanHossain @JobairHasan @rafee-adnan @Rajit13 Do you have any suggestions on this?

**Reply by rafee-adnan** [reply]
What was the context for the re-run again? Did you previously get any better fit on the arcs than this one? 
I remember from the last telecon, we discussed the possibility of excluding the satellite galaxy from the model. 
Would you kindly provide the context?

**Reply by aaimtiaz** [reply]
The gamma value was hitting the lower limit and the mass of the satellite galaxy was not contributing. Then I implemented the log_likelihood for theta_E and reran the MCMC. Today I have created a PR on this. #263

**Reply by SulymanHossain** [reply]
I think, your confusion has been resolved from today's meeting discussion. Let us know, if you have other doubts.

**Reply by aaimtiaz** [reply]
No, I don't have any other doubts.
